### PR TITLE
dev: Add colors & type styles to Storybook

### LINKED ISSUE #372: Add SFDS type & colors to Storybook
Add examples & docs implementing our chosen type styles and color swatches to Storybook

### PR DESCRIPTION
**Pull request by suzubara**
## Description 

This PR adds our font files and color swatches to new Storybook pages: **Global > Colors, Global > Typography, Global > Buttons**

Fixes #372 

## Review Notes

Start Storybook (`yarn storybook`) and navigate to the new pages. Review & verify that the colors and styles look accurate to our chosen styles.

image

image

image

image

### CODE REVIEW

**Comment by suzubara** [comment]
Awaiting feedback:
- link hover states?
- buttons?

**Review by jbecker01 — DISMISSED** [review]
👍

File: src/stories/colors.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  <li
+    style={{
+      borderRadius: '5px',
+      border: '1px solid lightgray',
+      padding: '8px',
+      marginBottom: '12px',
+    }}>
+    <span
+      style={{
+        backgroundColor: colors[color],
```
Comment (noahfirth) [inline src/stories/colors.stories.tsx:26]: Do we want to make this: `backgroundColor: colors[parseInt(color)],` to clear up this warning? It actually doesn't make a huge difference in Storybook to have the ability to pass a non-integer array index; these warnings always irritate me.

File: src/stories/colors.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        display: 'block',
+        height: '4em',
+        marginBottom: '0.3em',
+        borderRadius: '5px',
+      }}
+    />
+    <span style={{ fontFamily: 'monospace, monospace' }}>
+      ${color}
+      <br />
+      {colors[color]}
```
Comment (noahfirth) [inline src/stories/colors.stories.tsx:36]: Same as above; do we want to make this: `{colors[parseInt(color)]}` to clear up this warning?

**Review by noahfirth — COMMENTED** [review]
Everything looks good; not sure how we want to handle the errors about the potential to pass a non-integer array index in the `src/stories/colors.stories.tsx`. It doesn't truly matter for Storybook if we want to just let it ride. If we do, let me know and I can approve it!

**Review by malworks — APPROVED** [review]

**Comment by suzubara** [comment]
> Everything looks good; not sure how we want to handle the errors about the potential to pass a non-integer array index in the `src/stories/colors.stories.tsx`. It doesn't truly matter for Storybook if we want to just let it ride. If we do, let me know and I can approve it!

@noahfirth these values are strings so easy enough to parse & resolve the warning!

**Review by noahfirth — APPROVED** [review]
Awesome! Looking good!

**Pull request merged**